pageserve/app.py

```diff
@@ -19,7 +19,12 @@
 
 from flask import Flask, render_template, request, Response
 
-app = Flask(__name__)
+app = Flask(__name__)  # pylint: disable=invalid-name
+
+app.config["GAUTH_CLIENT_ID"] = os.environ.get("GAUTH_CLIENT_ID")
+# TODO(mukobi) make env var $USERS_ENDPOINT stop at /v1 (remove /authorization)
+# Makes this cleaner after cmei444 merges PR #26
```

Comment:
very minor note but my username has another 4 (cmei4444)

Reply:
Done https://github.com/knative-portability/large-events/pull/28/commits/d6d43f6fe46f4ba4a6dfe5007a7e881c5dd71280 https://github.com/knative-portability/large-events/pull/28/commits/9c624a1fc42c33797c886136fd50ba00133aade7. I make the env var `USERS_ENDPOINT` stop at /v1/ (you had it at /v1/authorization) so that it could more easily be used for other things by adding routes onto the url (e.g. add 'authenticate' for authentication).